README.md

```diff
@@ -2,6 +2,31 @@
 
 This repository's [issue tracker](https://github.com/osrf/infrastructure/issues) is used to receive reports from the Gazebo and ROS community regarding problems with the infrastructure provided by Open Robotics.
 
+Managed projects include
+
+* [Gazebo Answers](https://answers.gazebosim.org)
+* [Gazebo Community](https://community.gazebosim.org)
+* [Gazebo web site](https://gazebosim.org)
+* [Ignition web site](https://ignitionrobotics.org)
+* [OSRF Build farm](https://build.osrfoundation.org)
+* [OSRF package repositories](https://packages.osrfoundation.org)
+* [ROS 2 Build farm repositories](https://repo.ros2.org)
+* [ROS 2 Build farm](https://build.ros2.org)
+* [ROS 2 CI](https://ci.ros2.org)
+* [ROS Answers](https://answers.ros.org)
+* [ROS Build farm repositories](https://repositories.ros.org)
+* [ROS Build farm](https://build.ros.org)
+* [ROS Discourse](https://discourse.ros.org)
+* [ROS bootstrap repositories](https://repos.ros.org)
+* [ROS package repositories](http://packages.ros.org)
```

Comment:
```suggestion
* [ROS mailing lists](http://http://lists.ros.org)
* [ROS package repositories](http://packages.ros.org)
* [ROS planet web site](https://planet.ros.org/)
* [ROS robots web site](https://robots.ros.org/)
```

Reply:
I'd skip this one. It's only legacy/archive now, and been that way for many years.

navigation and moveit are on the cusp. We don't really maintain them and their in github pages, but I guess we could have them here and then forward on.